### DISCUSSION TITLE
Adaptation des fonctionnalités ma cantine à l'outre-mer 🌅

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1150] [Retours utilisateur·rice]
Il me semble que des modifications avaient déjà été faites, et, après un entretien avec le département/la région Réunion, on a de nouveaux retours / besoins : 

- **Notifier les seuils** outre-mer sur les cartes dans la **publication**.  En liant le lieu géographique à une information précise ?
- **Notifier les seuils** outre-mer sur l'onglet statistique. Ajout sigle "info" suivant les territoires avec un lien vers l'article par exemple ? 
- **Notifier les seuils** outre-mer sur les **affiches** générées dans ma cantine. Solution similaire à celle de publication ?

Les seuils EGAlim pour 2022-2025 oscillent entre 5% et 20% contre 50% au niveau national, et y a un besoin de contextualisation afin de ne pas donner l'impression au parent d'être "en retard" par rapport au territoire hexagonal.

**Comment by gauthierandre** [comment]
@alemangui @hfroot qu'en pensez-vous ?

**Reply by hfroot** [reply]
Perso j'ai rien à ajouter, je pense que c'est une bonne idée :)

**Comment by ghost** [comment]
Dans le même ordre d'idées, une utilisatrice de La Réunion m'a fait remarquer que les taux de produits bio et de produits durables adaptés aux DROM ne sont pas mentionnés sur la page "Mesures phares". @valeriemerle et @JenniferStephan, allons-nous mentionner ces taux sur cette page à terme?

**Reply by hfroot** [reply]
Pour l'instant on a donné un lien vers l'éxplication des seuils, est-ce tu trouves qu'on peut clarifier le texte ?

E.g. : (https://ma-cantine.beta.gouv.fr/mesures-phares/qualite-des-produits)

![Screenshot 2022-04-15 at 16 05 21](https://user-images.githubusercontent.com/9282816/163580505-d64819f1-fbdb-4a23-9228-970a7f9eea37.png)

**Reply by gauthierandre** [reply]
ça apparait où ça @hfroot ? Est-ce que ça peut pas apparaitre sur le diagnostic ? La page de publication ? Un peu comme ce dont on avait discuté juste au-dessus ?

> Il me semble que des modifications avaient déjà été faites, et, après un entretien avec le département/la région Réunion, on a de nouveaux retours / besoins :
> 
> * **Notifier les seuils** outre-mer sur les cartes dans la **publication**.  En liant le lieu géographique à une information précise ?
> * **Notifier les seuils** outre-mer sur l'onglet statistique. Ajout sigle "info" suivant les territoires avec un lien vers l'article par exemple ?
> * **Notifier les seuils** outre-mer sur les **affiches** générées dans ma cantine. Solution similaire à celle de publication ?
> 
> Les seuils EGAlim pour 2022-2025 oscillent entre 5% et 20% contre 50% au niveau national, et y a un besoin de contextualisation afin de ne pas donner l'impression au parent d'être "en retard" par rapport au territoire hexagonal.

**Reply by ghost** [reply]
@hfroot, l'article est clair.
Mais 2 suggestions :
- plutôt que "autres régions", écrire "départements et régions d'outre-mer (DROM)". Je pense que c'est plus explicite.
- mentionner les seuils directement sur la page Mesures phares. Si vous craignez d'alourdir cette page avec ces infos, pourquoi ne pas créer un menu déroulant "Seuils adaptés aux DROM" qui ne se déroule que quand on clique dessus?
